### PR TITLE
Hw 37

### PR DESCRIPTION
**Pull request by p1q**
ExecutorService/ForkJoin

### CODE REVIEW

File: src/main/java/multithreading/race/Main.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package multithreading.race;
```
Comment (Sofasmile) [inline src/main/java/multithreading/race/Main.java:1]: in this PR previous hw shouldn't be

File: src/main/java/multithreading/listsum/SumByForkJoin.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package multithreading.listsum;
+
+import java.util.List;
+import java.util.concurrent.RecursiveTask;
+
+public class SumByForkJoin extends RecursiveTask<Integer> {
+    private List<Integer> elements;
+    private Integer sum = 0;
+
+    SumByForkJoin(List<Integer> elements) {
```
Comment (Sofasmile) [inline src/main/java/multithreading/listsum/SumByForkJoin.java]: add access modifier

**Review by Sofasmile — APPROVED** [review]

File: src/main/java/multithreading/listsum/SumByForkJoin.java (hunk 2 above)
Comment (p1q) [reply src/main/java/multithreading/listsum/SumByForkJoin.java]: Done

File: src/main/java/multithreading/race/Main.java (hunk 1 above)
Comment (p1q) [reply src/main/java/multithreading/race/Main.java:1]: Got it. We spoke about merging these home works in the beginning.

**Pull request merged**